apps/api/src/app/events/events.module.ts

```diff
@@ -31,7 +31,7 @@ const PROVIDERS = [GetNovuProviderCredentials, StorageHelperService, CommunityOr
     ContentTemplatesModule,
     IntegrationModule,
     ExecutionDetailsModule,
-    TopicsModule,
+    TopicsV1Module,
```

Comment:
i wonder why TopicsV1Module is needed in the events module and TopicsV2Module is not. 
perhaps we can remove TopicsV1Module?

Reply:
There was some weird error, but I figured it out. Can be removed now 💪